### ISSUE TITLE
'High Contrast Black' theme Product Switch states WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1014]
**Describe the bug**
The Product Switch active and hover state color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/product-switch.html
2. Check Product Switch active and hover states' color contrast 

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81755981-49e13c80-946f-11ea-87d7-cf0295a33f30.png)
![image](https://user-images.githubusercontent.com/43764832/81756110-a93f4c80-946f-11ea-96b9-778183110978.png)

![image](https://user-images.githubusercontent.com/43764832/81753968-da1c8300-9469-11ea-8acc-3bd4f2de9a25.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)
